example-monorepos/with-custom-font/apps/next/pages/_document.tsx

```diff
@@ -55,7 +70,7 @@ export async function getInitialProps({ renderPage }) {
   return { ...page, styles: React.Children.toArray(styles) }
 }
 
-export class Document extends NextDocument {
+export default class Document extends NextDocument {
```

Comment:
Thanks for the quick merge @nandorojo!

I think line 91 also exports this as `default` 🙃